tests/needs-more-verification.test.js

```diff
@@ -22,8 +22,8 @@ describe('Third-party template needs more verification', () => {
         const npmPackage = '@company1/app-builder-template';
         const script = '../src/needs-more-verification.js';
         process.argv = ['node', script, npmPackage];
-        jest.isolateModules(() => {
-            require(script);
+        jest.isolateModules(async () => {
```

Comment:
`needs-more-verification.js` doesn't have any async operations, need to remove `async` there